### PR TITLE
Version bump (WIP)

### PR DESCRIPTION
**Pull request by tyarkoni**
This PR houses all remaining changes before we bump version to 0.2:

- [x] Add Results section to doc
- [x] Revise README and link to the Sphinx docs
- [x] Fixes for any bugs caught in the process of generating docs
- [x] Encrypt Google API access credentials on travis-ci
- [x] Add quickstart guide to docs

### CODE REVIEW

**Comment by tyarkoni** [comment]
Nice--with all of the keys properly installed, we're at 92% coverage!

**Comment by tyarkoni** [comment]
Okay @qmac, I think this should be ready, assuming the tests pass. If you get a chance to look this over in the next few hours that would be great (there aren't many changes to the code, it's almost all docs), but I'm pretty antsy to get this out, so I'll probably push the new release later tonight either way.

**Comment by qmac** [comment]
Ok, I'll skim through it in the next hour

**Comment by tyarkoni** [comment]
Ack, bunch of failures. Guess I forgot to patch the tests. Anyway, don't worry about the tests, I'm fixing them now. Just let me know if you catch any issues with the code or docs.

File: pliers/stimuli/text.py [hunk 1]
```diff
@@ -213,4 +217,5 @@ def tokenize_text(text):
         # TODO: track order as a separate attribute from duration, because we
         # can't treat serial position as if it were time in seconds.
```
Comment (qmac) [inline pliers/stimuli/text.py]: Can probably get rid of this comment now

**Comment by qmac** [comment]
Everything looks good. In regards to the `order` attribute we probably want to set it in `TextFilter`s as well (mostly just propagating).

**Comment by qmac** [comment]
Also, you can cherry pick 36e7bb6, I just put in a line about the API change for `ExtractorResult`s

File: pliers/tests/extractors/test_text_extractors.py [hunk 2]
```diff
@@ -24,8 +24,9 @@ def test_text_extractor():
                              variables=['length', 'frequency'])
     assert td.data.shape == (7, 2)
     result = td.transform(stim)[2].to_df()
-    assert result.iloc[0, 1] == 1
-    assert result.shape == (1, 5)
+    print(result)
```
Comment (qmac) [inline pliers/tests/extractors/test_text_extractors.py]: Dangling print

**Pull request merged**